tools/perf/ResultsComparer/Bdn/BdnComparer.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using ResultsComparer.Core;
+using Perfolizer.Mathematics.SignificanceTesting;
+using Perfolizer.Mathematics.Thresholds;
+
+namespace ResultsComparer.Bdn
+{
+    /// <summary>
+    /// Results comparer for BenchmarkDotNet full JSON reports.
+    /// </summary>
+    internal class BdnComparer : IResultsComparer
+    {
+        private const string FullBdnJsonFileExtension = "full.json";
+
+        public Task<bool> CanReadFile(string path)
+        {
+            bool isJson = path.EndsWith(".json");

```

Comment:
Is `.JSON` a possibility?

Reply:
I changed to a case-insensitive comparer